pytest_html/plugin.py

```diff
@@ -612,15 +615,57 @@ def _save_report(self, report_content):
             with open(style_path, "w", encoding="utf-8") as f:
                 f.write(self.style_css)
 
+    def _post_process_reports(self):
+        for test_name, test_reports in self.reports.items():
+            outcome = "passed"
+            wasxfail = False
+            failure_when = None
+            full_text = ""
+            extras = []
+            duration = 0.0
+
+            # in theory the last one should have all logs so we just go
+            #  through them all to figure out the outcome, xfail, duration,
+            #    extras, and when it swapped from pass
+            for test_report in test_reports:
+                full_text += test_report.longreprtext
```

Comment:
Can this ever be `None`? I ask since you add the following on line 278: `text = report.longreprtext or report.full_text`. Should we use `getattr` like we do below?